docs/guides/integration/open-from-github.md

```diff
@@ -111,26 +111,30 @@ https://stackblitz.com/github/vercel/next.js/tree/canary/examples/hello-world?ti
 
 ### Launching a script on project load
 
-Usually, in your project’s `package.json` file there is a script that you would instruct your users to run in order to, for instance, launch a development server. Suppose your `package.json` includes such `dev` script:
+StackBlitz will look in your project’s `package.json` file for a [npm script](https://docs.npmjs.com/cli/v8/using-npm/scripts) to run on project load. By default, it looks for a script named `"dev"` first, then for a script named `"start"`.
```

Comment:
Rewrote to describe the default behavior first, then options for changing that behavior.